build-tools-internal/src/main/groovy/elasticsearch.runtime-jdk-provision.gradle

```diff
@@ -20,17 +20,14 @@ import org.elasticsearch.gradle.testclusters.StandaloneRestIntegTestTask
 
 
 configure(allprojects) {
-    apply plugin: 'elasticsearch.jdk-download'
-
-    jdks {
-        provisioned_runtime {
-            vendor = VersionProperties.bundledJdkVendor
-            version = VersionProperties.bundledJdkVersion
-            platform = OS.current().name().toLowerCase()
-            architecture = Architecture.current().name().toLowerCase()
-        }
+    def launcher = javaToolchains.launcherFor {
```

Comment:
Is this guaranteed to be the exact version specified in `version.properties`? How do we account for the fact that we can only request by major version?